### ISSUE TITLE
Issues hit when integrating with starfish

### ISSUE THREAD

**Issue by jkozlowski** [jkryl/spdk-sys#1]
Will keep a list here and send PRs once finished:

* need `sudo ./scripts/pkgdep.sh` or similar to install spdk deps automatically.
* isa-l dep is optional in spdk, enabled by default for x86 only. The bigger thing is that it requires latest nasm, which wasn't available on Ubuntu 18.04 - I had upgrade. The `build.sh` script should probably be resilient if this library is not built.
* Had to manually run `sudo make install` in spdk subdir to get headers to be installed, so that the rust `build.rs` can find them.

**Comment by jkryl** [comment]
I agree. This should be fixed. Just a note about x86. Linux x86_64  is actually the only platform supported by the build script currently. cross compilation issues are getting just now addressed by spdk team. Hopefully with the next spdk version it will be easier to build spdk lib which works on all recent x86 processors regardless which exact type of cpu is used.
